src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+import uuid
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        max_length=40,
+        help_text=_(
+            "Unieke (technische) identificatiecode van de betrokkene bij klantcontact."
+        ),
+    )
+    nummer = models.PositiveIntegerField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[MaxValueValidator(9999999999)],
+        blank=False,
+    )
+    interne_notitie = models.CharField(
+        _("Interne notitie"),
+        help_text=_(
+            "Mededelingen, aantekeningen of bijzonderheden over de partij, bedoeld voor intern gebruik."
+        ),
+        max_length=1000,
+        blank=True,
+    )
+    soort_partij = models.CharField(
+        _("Soort partij"),
+        help_text=_("Geeft aan van welke specifieke soort partij sprake is."),
+        max_length=14,
+        choices=SoortPartij.choices,
+        blank=False,
+    )
+    indicatie_geheimhouding = models.BooleanField(
+        _("Indicatie geheimhouding"),
+        help_text=_(
+            "Geeft aan of de verstrekker van partijgegevens heeft aangegeven dat "
+            "deze gegevens als geheim beschouwd moeten worden."
+        ),
+        blank=False,
+    )
+    voorkeurskanaal = models.CharField(
+        _("Voorkeurskanaal"),
+        help_text=_(
+            "Kanaal dat de partij bij voorkeur gebruikt voor contact met de gemeente."
+        ),
+        max_length=50,
+        blank=True,
+    )
+    # TODO: change max_length to more realistic number
+    voorkeurstaal = models.CharField(
+        _("Voorkeurstaal"),
+        help_text=_(
+            "Taal waarin de partij bij voorkeur contact heeft met de gemeente."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    indicatie_actief = models.BooleanField(
+        _("Indicatie actief"),
+        help_text=_(
+            "Geeft aan of de contactgegevens van de partij nog gebruikt morgen worden om contact op te nemen. "
+            "Gegevens van niet-actieve partijen mogen hiervoor niet worden gebruikt."
+        ),
+        blank=False,
+    )
+
+    # Correspondentieadres model fields:
+    correspondentieadres_nummeraanduiding_id = models.UUIDField(
```

Comment:
This is now a required field, but the group 'correspondentieadres' is not required (however, if there is a correspondentieadres, the id is required).

Reply:
Wijzig naar CharField(255) met een # TODO er bij dat dit nog opgehelderd moet worden.